mule-extensions-api/src/main/java/org/mule/runtime/extension/api/OnTerminateInformation.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.runtime.extension.api;
+
+import static java.util.Optional.ofNullable;
+import org.mule.runtime.api.message.Error;
+import org.mule.runtime.extension.api.runtime.source.SourceCallbackContext;
+
+import java.util.Optional;
+
+public class OnTerminateInformation {
```

Comment:
Horrible name, no jdoc, doesn't seem used, at least not in the http PR. Is this for internal use? If so, this doesn't belong here